src/_includes/partials/token_table.njk

```diff
@@ -0,0 +1,58 @@
+<table class="table-display-token-{{token}}">
+<tr>
+{% for header in headers %}
+<th>{{ header[1] }}</th>
+{% endfor %}
+</tr>
+{% comment %}
+tokens: all the tokens
+token: key
+filteredTokens: the filtered tokens
+subTokens: part of the filtered tokens
+{% endcomment %}
+{% assign filteredTokens = token | split: "." | dig: tokens['Tokens'] %}
```

Comment:
This allows us to recursively dig into the token tree if we want. we simply pass in a string like this: "token.dig.into.the.tree" from the partial